files/zh-cn/web/api/ui_events/keyboard_event_key_values/index.md

```diff
@@ -3074,7 +3074,7 @@ The values below are derived in part form a number of consumer electronics techn
 
 - [DTV Application Software Environment](http://atsc.org/standard/a100-dtv-application-software-environment-level-1-dase-1/) (part of the [ATSC](https://zh.wikipedia.org/wiki/ATSC) specification)
 - [Open Cable Application Platform 1.1.3](http://www.cablelabs.com/specifications/OC-SP-OCAP1.1.3-100603.pdf)
-- [ANSI/CEA-2014-B](http://www.ce.org/Standards/Standard-Listings/R7-Home-Network-Committee/CEA-2014-B-(ANSI%29.aspx): Web-based Protocol and Framework for Remote User Interface on UPnP™ Networks and the Internet
+- [ANSI/CEA-2014-B](<http://www.ce.org/Standards/Standard-Listings/R7-Home-Network-Committee/CEA-2014-B-(ANSI%29.aspx>): Web-based Protocol and Framework for Remote User Interface on UPnP™ Networks and the Internet
```

Comment:
let's update this link (https://github.com/mdn/content/blob/db9797df4277b7f4c004f1a2b079b1e2f87bc8cb/files/en-us/web/api/ui_events/keyboard_event_key_values/index.md?plain=1#L3077):

```suggestion
- [ANSI/CEA-2014-B](https://shop.cta.tech/products/web-based-protocol-and-framework-for-remote-user-interface-on-upnp-networks-and-the-internet): Web-based Protocol and Framework for Remote User Interface on UPnP™ Networks and the Internet
```